### ISSUE TITLE
Idea's and 1.11

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#444]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/443

**Originally posted by Darkman_Bree (Nov 17, 2016):** 

Hello developers of Shopkeepers.My name is Darkman_Bree, I LOVE your plugin, I use it alot in my server, and I wanna give some suggestions for the hopefully upcoming version of 1.11 Shopkeepers.To start with: Minecraft 1.11 just released and the plugin needs to be compatible with it.Now the idea's:I have seen replies of people asking how to add more than 8 trades, I got an idea:This is a picture of how it looks like:http://imgur.com/a/0sZOhOn the picture, You can see a normal trade on the left (nothing special), On the right, the regular naming, profession and delete, But next to it, there are 3 new items, Those are pages.The Red Stained Glass Pane will turn a page back. (Page 2 to Page 1, Cannot go under 1.)
The Lime Stained Glass Pane will turn a page forward. (Page 1 to Page 2, Limit can be changed in the configs)
The Sunflower is renamed to the current page you are in: PAGE 1And my last idea: Since 1.11 adds the Green Villager back, can you add it back to the professions?Thank you for reading!
-Darkman_Bree

**Comment by blablubbabcDEV** [comment]
**Originally commented by blablubbabc (Nov 17, 2016):** 

I added a separate ticket for the 'more than 8 trades' request (https://dev.bukkit.org/bukkit-plugins/shopkeepers/tickets/445-allow-setup-of-more-than-8-trades/). Though I don't expect me to find time to look into this in the near future.
Regarding green villager: It looks like spigot hasn't yet added that to there 1.11 version of the bukkit api yet. I created a spigot ticket asking for that to be added.

**Comment by blablubbabcDEV** [comment]
**Originally commented by Darkman_Bree (Nov 17, 2016):** 

[at]blablubbabc:
Okay thank you, I also wanna say.
I ran a 1.11 server with Shopkeepers, I was able to create one, add trades, but I couldn't open the trades.

**Comment by blablubbabcDEV** [comment]
**Originally commented by blablubbabc (Nov 18, 2016):** 

[at]Darkman_Bree:
Yeah, the 'compatibility mode' fallback, which is used in case Shopkeepers is run on a non-supported (new) minecraft version, seemed to be slightly outdated. I updated it, so it hopefully works on the next minecraft update then. However, I don't suggest running with this because of the risk for things to break depending on the changes of the minecraft update.
I was able to work on a proper update for MC 1.11 today, and will probably upload that tomorrow.
